pkg/datastore/apikey.go

```diff
@@ -35,6 +36,21 @@ func (a *apiKeyCollection) Factory() Factory {
 	}
 }
 
+func (a *apiKeyCollection) GetStoredFileNames(id string) []string {
+	return []string{id}
+}
+
+func (a *apiKeyCollection) GetUpdatableFileName(id string) (string, error) {
```

Comment:
How about passing the commander as an argument of this function to use instead of from a field of collection struct?

Reply:
I think about it at first, in that case, we have to pass the commander via database `Update` interface. Tbh, I understand your point since passing commander via collection is a bit odd to me as well, but when I think about it clearly, the database layer (filedb implementation in this case) does not really care about which commander request this operation, just want to know which file should be called to handle request from this commander. So, I think the commander only has value to the collection (to know who is the commander for this collection at that time), I guess there is no reason to change the `Update` datastore interface to pass commander directly, wdyt?